README.md

```diff
@@ -162,6 +162,14 @@ $.Date :: Type
 
 Type comprising every Date value.
 
+#### `Enum`
+
+```haskell
+$.Enum :: [Any] -> Type
+```
+
+Constructor for [enumerated types][2].
+
```

Comment:
Also, for consistency with `$.RecordType` we should name this `$.EnumType`. I realize that `$.Nullable` is the odd one out, but my thinking was that there's not much value in writing `const NullableString = $.NullableType($.String);` since one could simply write `$.NullableType($.String)` in situ. At that point it would be nice to be able to write `$.Nullable($.String)` instead, as it reads better. For enums and records, though, there's clearly an advantage to defining the type in just one place and reusing the definition.